eng/Versions.props

```diff
@@ -48,6 +44,13 @@
     <!-- Packages from dotnet/arcade -->
     <MicrosoftDotNetGenAPIPackageVersion>1.0.0-beta.19309.1</MicrosoftDotNetGenAPIPackageVersion>
   </PropertyGroup>
+  <PropertyGroup>
```

Comment:
```suggestion
  <PropertyGroup Label="Manual" >
```

Reply:
Oops, I see now. Why not just put this version in the manual versions?

Reply:
It's not really a manual package version either. Maybe 'Label="Dependency version settings"'?

Reply:
Sure. It's super nitpicky anyways. I just want us to have the same changes.

Reply:
Ok, updated to add a label.